js/apps/system/_admin/aardvark/APP/aardvark.js

```diff
@@ -1642,7 +1680,7 @@ authRouter.get('/graphs-v2/:name', function (req, res) {
         bindToWindow: false
       },
       multiselect: false,
-      navigationButtons: true,
+      navigationButtons: false,
```

Comment:
Does this have any effect, because currently I don't see any navigation buttons

Reply:
The import of the js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/vis-network.css would make them appear now in the Graph Viewer. And as we don't plan to display them `navigationButtons: false` hides them.